### ISSUE TITLE
Ensure LMS User Flow consistent with Google Classroom

### ISSUE THREAD

**Issue by thebradbain** [education/classroom#2029]
**Action Items**
 
- [x] Move initial LMS Configuration setup flow into the Classroom Profile (Issue #2050)
- [x] When creating a new classroom, ask for the user about integration at the beginning
- [x] Remove dedicated LMS Configuration page from sidebar 
- [x] Refer to the LMS by its official platform name on the front-end (e.g. Canvas, Blackboard, Moodle) rather than generic LMS/Learning Management System (Issue #2057, PR #2056, PR #2071)

**Further Context**

After speaking with @brandonrosage, we ironed out some details on where the configuration of LMS should live, and how it should be treated alongside Google Classroom integration from the user's perspective.

In short, we should ensure the setup process is the same and common functionality is consistent and accessible in the same ways.

See #2027 for further context re: Google Classroom
